### ISSUE TITLE
Report contradictory proof statuses

### ISSUE THREAD

**Issue by tillmo** [ontohub/ontohub#1321]
For Ax2_20 of http://develop.ontohub.org/sandbox/Family2//FamilyBase///theorems, locally I got both THM and NOC. Such contradictory proof statuses should be reported at a special page, because they indicate a bug that should be fixed quickly.

**Comment by tillmo** [comment]
I just see that a time-out for SPASS leads to NOC. This is incorrect. Only a completion should lead to NOC.

**Comment by eugenk** [comment]
Right, now they are a bug, but later they could be result of different configurations (e.g. axiom selection).

**Comment by tillmo** [comment]
OK, but I meant a mechanism searching for bugs. That means that THM and NOC are contradictory only if the selected axioms are the same (or if NOC results from a selection that is larger than that for THM).
